src/core/analysis/mcmc/output/TraceNumeric.cpp

```diff
@@ -46,14 +46,14 @@ TraceNumeric* TraceNumeric::clone() const
 void TraceNumeric::computeStatistics( void )
 {
 
-    // test stationarity within chain
-    size_t nBlocks = 10;
-    StationarityTest testS = StationarityTest(nBlocks, 0.01);
-    passedStationarityTest = testS.assessConvergence(*this);
-
-    // Geweke's test for convergence within a chain
-    GewekeTest testG = GewekeTest(0.01);
-    passedGewekeTest = testG.assessConvergence(*this);
+//    // test stationarity within chain
```

Comment:
If nobody is using it, then maybe remove it?

Or, if somebody might fix it, then may be add a comment like.  Maybe some thing like:
"Sebastian: This code doesn't work any more and caused some runs to crash.
                    I'm commenting it out in case somebody wants to try and fix it."

Up to you.